Lazydocs: API Reference doc refresh (#1424)
wandb/docs: content/en/guides/models/track/log/working-with-csv.md

```

> 2. Next,  start a new W&B Run to track and log to W&B with [`wandb.init()`]({{< relref "/ref/python/sdk/functions/init" >}}):

Reviewer: ```suggestion
2. Next, start a new W&B Run to track and log to W&B with [`wandb.init()`]({{< relref "/ref/python/sdk/functions/init" >}}):
```

> ```python

Reviewer: This code block needs to be indented to beyond the N above.

run = wandb.init(
    project=PROJECT_NAME, name=run_name, tags=tags, notes=notes, config=config
)
```

> As an experiment runs, you might want to log every instance of your metrics so they are available to view, query, and analyze with W&B. Use the [`run.log()`]({{< relref "/ref/python/sdk/classes/run/#method-runlog" >}}) command to accomplish this:

Reviewer: Should these paragraphs also be numbered steps with the code blocks within them?

```python
run.log({key: val})
```

> You can optionally log a final summary metric to define the outcome of the run. Use the W&B  [`define_metric`]({{< relref "/ref/python/sdk/classes/run#define_metric" >}}) API to accomplish this. In this example case, we will add the summary metrics to our run with `run.summary.update()`:

Reviewer: ```suggestion
You can optionally log a final summary metric to define the outcome of the run using the [`define_metric`]({{< relref "/ref/python/sdk/classes/run#define_metric" >}}) API. This example adds the summary metrics to our run with `run.summary.update()`:
```
